api-v2/src/auth/jwt.strategy.ts

```diff
@@ -29,10 +29,12 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
    * @param jwtPayload decoded payload stored in the JWT
    */
   async validate(jwtPayload: JwtPayload): Promise<Student> {
-    try {
-      return await this.authService.validateJwtPayload(jwtPayload);
-    } catch (_error) {
+    const student = await this.authService.validateJwtPayload(jwtPayload);
+
+    if (!student) {
       throw new UnauthorizedException('Invalid JWT');
```

Comment:
thoughts on returning without a string here? i'm cautious of having good error messages for login, because of information it might leak to hackers

Reply:
Good point!